kansha/user/models.py

```diff
@@ -25,6 +25,21 @@
 from kansha.models import Entity
 
 
+class DataToken(Entity):
+    using_options(tablename='token')
+    token = Field(Unicode(255), unique=True, nullable=False, primary_key=True)
+    username = Field(Unicode(255), nullable=True)
+    date = Field(DateTime, nullable=False)
+    action = Field(Unicode(255), nullable=True)
+    board = ManyToOne('DataBoard')
+
+    @classmethod
+    def delete_token_by_username(cls, username, action):
```

Comment:
'delete_toke_by_username' -> 'delete_by_username' (we already are in the 'DataToken' class)